R/digestFastqs.R

```diff
@@ -512,6 +518,13 @@ digestFastqs <- function(fastqForward, fastqReverse = NULL,
     stop("'useTreeWTmatch' must be a logical scalar.")
   }
   
+  if (!is.logical(collapseToWTForward) || length(collapseToWTForward) != 1) {
+    stop("'collapseToWTForward' must be a logical scalar.")
+  }
+  if (!is.logical(collapseToWTReverse) || length(collapseToWTReverse) != 1) {
+    stop("'collapseToWTReverse' must be a logical scalar.")
+  }
+  
```

Comment:
I think this is again more cosmetical than really needed: We could use `.assertScalar` and `.assertVector` more for argument checking in `digestFastqs` and maybe elsewhere.

A special case is `mutscan:::checkNumericInput`: It can probably not be fully replaced by `.assertXXX`, because of the special case of allowing only values that are non-negative or -1.

Reply:
Yes, agree. I switched to using `.assertXXX` in many of the other functions, but not yet `digestFastqs`. I think it should work also to replace `checkNumericInput` with the latest version of the utils.

Reply:
Resolved in 9ce3c2b